lib/xdp_loader.h

```diff
@@ -0,0 +1,31 @@
+#ifndef __XDP_LOADER_H_
+#define __XDP_LOADER_H_
```

Comment:
There should be a consistent include guard style. Following the style of lib/ebpd.h we should have:
#define LIB_XDP_LOADER_H_

Note that once class XdpLoader is removed you may want to rename this file.